crypto/rsa/rsa_lib.c

```diff
@@ -259,6 +327,37 @@ void RSA_get0_factors(const RSA *r, const BIGNUM **p, const BIGNUM **q)
         *q = r->q;
 }
 
+size_t RSA_get_multi_prime_extra_count(const RSA *r)
+{
+    int pnum;
+
+    pnum = sk_RSA_PRIME_INFO_num(r->prime_infos);
+    if (pnum <= 0)
+        pnum = 0;
+    return (size_t)pnum;
+}
+
+int RSA_get0_multi_prime_factors(const RSA *r, const BIGNUM **oth_primes)
+{
+    size_t pnum;
+    RSA_PRIME_INFO *pinfo;
+    size_t i;
+
+    if ((pnum = RSA_get_multi_prime_extra_count(r)) == 0)
+        return 0;
+
+    /*
+     * return other primes
+     * it's the user's job to guarantee the buffer length
```

Comment:
It's not clear which buffer and what its length. May I suggest "it's caller's responsibility to allocate oth_primes[pnum]"?

Reply:
Changed